1-D_Array/Python/EvenOdd.py

```diff
@@ -0,0 +1,48 @@
+# Python program to count even and odd numbers in an array
+
+# To count number of even elements
+def  no_of_even_elem(arr):
+
+    # Initialization
+    even_num = 0
+
+    # Finding even numbers in array
+    for i in range (0,len(arr)):
+        if arr[i] %2 == 0:
+            even_num +=1
+
+    return even_num
+            
+
+#--- main--- function
+
+# initialising variables
+arrlength = 0
+num  = 0
+
+#input array from user
+
+lst = []    # creating an empty list 
+  
+n = int(input("Enter number of elements : ")) # number of elements of list 
```

Comment:
take this commend to the previous line

Reply:
@asha15  I modified the commit and made the required changes.